pkg/service/sip.go

```diff
@@ -364,24 +364,29 @@ func (s *SIPService) CreateSIPParticipant(ctx context.Context, req *livekit.Crea
 	}, nil
 }
 
-func (s *SIPService) CreateSIPParticipantRequest(ctx context.Context, req *livekit.CreateSIPParticipantRequest, wsUrl, token string) (*rpc.InternalCreateSIPParticipantRequest, error) {
+func (s *SIPService) CreateSIPParticipantRequest(ctx context.Context, req *livekit.CreateSIPParticipantRequest, projectID, host, wsUrl, token string) (*rpc.InternalCreateSIPParticipantRequest, error) {
 	if err := EnsureSIPCallPermission(ctx); err != nil {
 		return nil, twirpAuthError(err)
 	}
 	if s.store == nil {
 		return nil, ErrSIPNotConnected
 	}
 	callID := sip.NewCallID()
+	log := logger.GetLogger()
+	if projectID != "" {
+		log = log.WithValues("projectID", projectID)
+	}
+	log = log.WithValues(
+		"callId", callID,
+		"roomName", req.RoomName,
```

Comment:
in our logger context, it's always just room

```suggestion
		"room", req.RoomName,
```